packages/provider/src/lib/provider.service.ts

```diff
@@ -42,8 +42,7 @@ export class ProviderService {
     [FLAGD_PROVIDER_ID]: {
       factory: () => new FlagdProvider(),
       host: process.env.FLAGD_HOST_WEB ?? 'localhost',
-      // double NOT bitwise operator used to convert env to number or default
-      port: ~~(process.env.FLAGD_PORT_WEB ?? 8013),
+      port: Number.parseInt(process.env.FLAGD_PORT_WEB || '8013') || 8013,
```

Comment:
This handles if the env is falsy, or if it's not a valid number.

It does default silently however, but that seems like what we were doing before anyway.